scheduler.rst

```diff
@@ -0,0 +1,293 @@
+
+However, unlike Symfony Messenger, the messages will not be dispatched in the first instance. Instead, the aim is to create them based on a predefined frequency.
+
+This is where the specific transport in Symfony Scheduler, known as the :class:`Symfony\\Component\\Scheduler\\Messenger\\SchedulerTransport`, plays a crucial role. The transport autonomously generates directly various messages according to the assigned frequencies.
+
+From (Symfony Messenger cycle):
+
+.. image:: /_images/components/messenger/basic_cycle.png
+    :alt: Symfony Messenger basic cycle
+
+To (Symfony Scheduler cycle):
+
+.. image:: /_images/components/scheduler/scheduler_cycle.png
+    :alt: Symfony Scheduler basic cycle
+
+In essence, it is crucial to precisely define the message to be conveyed and processed. In Symfony Scheduler, the concept of a message takes on a very particular characteristic; it should be recurrent: It's a :class:`Symfony\\Component\\Scheduler\\RecurringMessage`.
+
+Clarifying the need to define and attach Recurring Messages to a Schedule.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to generate various messages based on their defined frequencies, configuration is necessary.
+The heart of the scheduling process and its configuration resides in a class that must extend the :class:`Symfony\\Component\\Scheduler\\ScheduleProviderInterface`.
+
+The purpose of this provider is to return a schedule through the method ``getSchedule()`` containing our different recurringMessages.
+
+The class attribute ``AsSchedule``, which by default references the ``default`` named schedule, allows us to register on a particular schedule::
+
+    // src/Scheduler/MyScheduleProvider.php
+    namespace App\Scheduler;
+
+    class MyScheduleProvider implements ScheduleProviderInterface
+    {
+        public function getSchedule(): Schedule
+        {
+            ...
+        }
+    }
+
+.. tip::
+
+    This becomes important when initiating the ``messenger:consume`` command, especially when specifying one or more specific transports.
+    In Symfony Scheduler, the transport is named ``scheduler_nameofyourschedule``.
+
+The Concept of Recurring Message in Symfony Scheduler
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A message associated with a Trigger
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First and foremost, a RecurringMessage is a message that will be associated with a trigger.
+
+The trigger is what allows configuring the recurrence frequency of your message. Several options are available to us:
+
+#. It can be a cron expression trigger:
+
+.. configuration-block::
+
+    .. code-block:: php
+
+        RecurringMessage::cron(‘*****’, new Message())
+        RecurringMessage::cron('#daily', $msg);
+
+.. tip::
+
+    A hashed expressions is a string representing the schedule for a particular command to execute.
```

Comment:
This part isn't quite correct. What you're describing is the standard cron aliases

- `@daily`
- `@monthly`
- etc...

Hashed aliases are different in that they create a random but deterministic standard cron expression based on some context.

At some point later in the doc, we need to discuss the importance of not having many tasks all run at the same time (ie 10 jobs at 00:00 every day). The jitter trigger wrapper is one solution, another is hashed cron expressions.

Why don't you drop this section for now (or just show more examples of _standard_ cron expressions) and I'll created a follow-up PR explaining hashed expressions. I originally ported this code to Symfony and use it quite extensively.